packages/danger/src/PullRequestManager.ts

```diff
@@ -59,63 +63,48 @@ export class PullRequestManager {
   parseChangelogSuggestionFromDescription(): ChangelogEntries {
     const changelogEntries = {
       [DEFAULT_CHANGELOG_ENTRY_KEY]: {
-        type: this.skip ? ChangelogEntryType.SKIP : ChangelogEntryType.NOT_INCLUDED,
-        message: this.pullRequest.title.replace(/\[.*\]/, '').trim(),
+        type: ChangelogEntryType.BUG_FIXES,
+        message: this.prTitle.replace(/\[.*\]/, '').trim(),
       },
     };
 
-    const parseLine: (line: string) => void = line => {
-      const parsingResult = this.parseTagsFromLine(line);
+    this.changelogSection.forEach(line => {
+      const { packageName, type } = this.parseTagsFromLine(line);
       const message = line.replace(/\[.*\]/, '').trim();
```

Comment:
Hmm, shouldn't it be 👇 ?
```suggestion
      const message = line.replace(/^(\[[^\]]\])+/g, '').trim();
```